From f813b4ef185acedcc450ca03a39bc7f49eafb260 Mon Sep 17 00:00:00 2001
From: Stefan Chulski <stefanc@marvell.com>
Date: Tue, 29 Mar 2016 15:15:36 +0300
Subject: [PATCH 0994/1240] dts: 7040-rz: Added 10G KR and 2.5G support

-Added phy-speed 2500 for SGMII and mac2 phy mode changed to sgmii
-In KR dts mac0 phy mode changed to KR
-armada-7040-rz-db_kr.dts mac0 phy-mode updated from sgmii to kr
-armada-7040-rz-db_router.dts phy-speed updated from PHY_SPEED_1_25G to
 PHY_SPEED_3_125G and mac2 phy-mode from rgmii to sgmii

Change-Id: I480a931b787553719dd99501db0998527aeaa766
Reviewed-on: http://vgitil04.il.marvell.com:8080/28631
Reviewed-by: Haim Boot <hayim@marvell.com>
Tested-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/dts/armada-7040-rz-db_kr.dts     | 2 +-
 arch/arm/dts/armada-7040-rz-db_router.dts | 5 +++--
 2 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/dts/armada-7040-rz-db_kr.dts b/arch/arm/dts/armada-7040-rz-db_kr.dts
index ed786ac..9b8aae6 100644
--- a/arch/arm/dts/armada-7040-rz-db_kr.dts
+++ b/arch/arm/dts/armada-7040-rz-db_kr.dts
@@ -159,7 +159,7 @@
 			};
 			gop {
 				mac0 {
-					phy-mode = "sgmii"; /* lane-2 */
+					phy-mode = "kr"; /* lane-2 */
 					status = "okay";
 				};
 				mac2 {
diff --git a/arch/arm/dts/armada-7040-rz-db_router.dts b/arch/arm/dts/armada-7040-rz-db_router.dts
index 3f53ff9..2bd33af 100644
--- a/arch/arm/dts/armada-7040-rz-db_router.dts
+++ b/arch/arm/dts/armada-7040-rz-db_router.dts
@@ -116,7 +116,7 @@
 			comphy {
 				phy0 {
 					phy-type = <PHY_TYPE_SGMII2>;
-					phy-speed = <PHY_SPEED_1_25G>;
+					phy-speed = <PHY_SPEED_3_125G>;
 				};
 				phy1 {
 					phy-type = <PHY_TYPE_USB3_HOST0>;
@@ -155,8 +155,9 @@
 					status = "okay";
 				};
 				mac2 {
-					phy-mode = "rgmii"; /* rgmii-0 */
+					phy-mode = "sgmii"; /* rgmii-0 */
 					phy = <&phy2>;
+					phy-speed = <2500>;
 					status = "okay";
 				};
 				mac3 {
-- 
1.9.1

